### PR TITLE
Add raw data before fixes in gift format

### PR DESCRIPTION
**Pull request by karlsve**

### CODE REVIEW

**Comment by leukipp** [comment]
Found an easy way to fix, or at least improve the positions of wrong coordinates.

Use the [original coordinates](https://github.com/standupmaths/xmastree2021/blob/ca8f2b88213a1750b06c6e69f22280def777a76a/light_fixer.py#L2) and remove the "noise" (e.g. with signal noise filtering, line fitting methods, ML algorithms) for each axis, since such a large variation should not be possible due to cable length limitations.

```python
import numpy as np
from sklearn.neighbors import KNeighborsRegressor

def fix_it(coords, k=7):
    """ coords of type numpy array with shape (500, 3) """
    idx = np.arange(coords.shape[0]).reshape(-1, 1)
    knr = KNeighborsRegressor(n_neighbors=k, weights='uniform')
    return knr.fit(idx, coords).predict(idx)

coords_raw = np.array([[152, -93, -233], [170, -282, 228], ...])
coords_fixed = fix_it(coords_raw)
```
Here the raw coordinates are shown and the code above produces the fixed coordinates in gray by considering 7 neighbors in the chain.
coords

This is how the resulting tree looks when the coordinates have been cleaned up.
tree

Here I've left the green coordinates as determined by @thatmattparker `light_fixer.py` script (using the 95px distance threshold) and just replaced all the outliers with the KNeighborsRegressor estimate.

Would be interesting to know if this would be indeed a better presentation of the ground truth coordinates.
